### PR TITLE
Update instruction on an existing Rails app

### PR DESCRIPTION
**Pull request by ahangarha**
Updated the instruction to refer to `shakapacker` rather `webpacker`.

I recommend merging #1491 before this instruction and updating this guide based on those changes otherwise, we need to make a new PR.

- - -
This change is [Reviewable](https://reviewable.io/reviews/shakacode/react_on_rails/1495)

### CODE REVIEW

**Review by Judahmeek — APPROVED** [review]

File: docs/guides/installation-into-an-existing-rails-app.md [hunk 1]
```diff
@@ -7,48 +7,45 @@
...
-_Use the latest version for react_on_rails._
+   Or use `bundle add`:
 
-```
-bundle add webpacker                 
-bundle add react_on_rails --version=12.0.4 --strict
-```
+   ```bash
+   $ bundle add react_on_rails --version=13.1.0 --strict
+   $ bundle add shakapacker
```
Comment (justin808) [inline docs/guides/installation-into-an-existing-rails-app.md]: @ahangarha what is the `$` for? your personal prompt?

Comment (ahangarha) [reply docs/guides/installation-into-an-existing-rails-app.md]: Thanks for bringing this up. This was one of my concerns and since it was minor, I didn't bring it up. In our docs, sometimes we have this `$` sign (which represents the shell of a non-root user). To me, not only this is not necessary (the user should know which code is to run in the shell and which one to is to add to a file), but it is annoying especially when one wants to copy/paste the code block.

It should be enough to have syntax highlighting.

Now I am confident to remove `$` from all such cases.

Comment (ahangarha) [reply docs/guides/installation-into-an-existing-rails-app.md]: @justin808 Done

**Comment by justin808** [comment]
@ahangarha please resolve conflicts.

File: spec/dummy/Gemfile.lock [hunk 2]
```diff
@@ -11,67 +11,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (7.0.0)
-      actionpack (= 7.0.0)
-      activesupport (= 7.0.0)
+    actioncable (7.0.4.2)
```
Comment (justin808) [inline spec/dummy/Gemfile.lock]: gem version items should be in separate PR!

**Review by justin808 — CHANGES_REQUESTED** [review]

File: spec/dummy/Gemfile.lock (hunk 2 above)
Comment (ahangarha) [reply spec/dummy/Gemfile.lock]: @justin808 I removed the commit. I am not sure why I pushed that commit.
